core/const.py

```diff
@@ -870,3 +870,31 @@ class ITLRegion(StrEnum):
         "RiskOwnerRole": "P{i}",
     },
 }
+
+# maps the financial year's start tear, or a month, back to original col letter
+# non-footfall outcomes are mapped by the financial year's start year, but footfall outcomes just by the month
+# TODO: add non-outcome tables to this
+FINANCIAL_YEAR_OR_MONTH_TO_ORIGINAL_COLUMN_LETTER = {
+    "2020": "F{i}",
+    "2021": "G{i}",
+    "2022": "H{i}",
+    "2023": "I{i}",
+    "2024": "J{i}",
+    "2025": "K{i}",
+    "2026": "L{i}",
+    "2027": "M{i}",
+    "2028": "N{i}",
+    "2029": "O{i}",
+    "April": "D{i}",
+    "May": "E{i}",
+    "June": "F{i}",
+    "July": "G{i}",
+    "August": "H{i}",
+    "September": "I{i}",
+    "October": "J{i}",
+    "November": "K{i}",
+    "December": "L{i}",
+    "January": "M{i}",
+    "February": "N{i}",
+    "March": "O{i}",
+}
```

Comment:
Does it make sense to have all of these dates in the same dictionary - could it be clearer to have two separate dictionaries with more accurate names? 

This gives the sense that it's generic but is actually very specific. 

There is a TODO comment to add non-outcome tables but I think that will mean this is more unclear?

Reply:
I think it might make more sense to have a dictionary mapping per table that needs this extra layer of mapping as per Connor's comment, but I'll do a quick investigation to see if there's a reasonable way we can get all of the tables requiring this mapping to only use one mapping or not.

Reply:
Worth considering if it's clear enough to have them in one table, even if it's feasible with unique keys